### ISSUE TITLE
Scheduled dark mode.

### ISSUE THREAD

**Issue by baduhai** [getferdi/ferdi#805]
**Is your feature request related to a problem? Please describe.**
I have my system set up to switch between light and dark themes depending on time, it'd be nice if Ferdi could comply with that.

**Describe the solution you'd like**
There could be an option, in addition to enabling dark mode, to schedule dark mode; i.e. dark mode from 20:00 to 06:00. The functionality is already there for scheduled do-not-disturb, so I'd imagine this is simple to implement.

**Comment by nordurljosahvida** [comment]
> Synchronize dark mode with my OS's settings

@vantezzen where is the "Synchronize dark mode with my OS's settings" you're talking about? Is this the same as what I'm suggesting here? https://github.com/getferdi/ferdi/issues/833

Thanks!

**Comment by nordurljosahvida** [comment]
ok, just realized it's already available in win and mac, this is amazing, opened thread for linux here: #833

**Comment by baduhai** [comment]
> > Synchronize dark mode with my OS's settings
> 
> @vantezzen where is the "Synchronize dark mode with my OS's settings" you're talking about? Is this the same as what I'm suggesting here? https://github.com/getferdi/ferdi/issues/833
> 
> Thanks!

Yes and no. I'm suggesting that and the ability to schedule it through Ferdi. So you'd have two possibilities:

1. Ferdi follows the OS schedule to switch between light and dark.
2. Ferdi follows its own schedule to switch between light and dark. In this option Ferdi could make use of the code already written for the scheduled do-not-disturb feature. So this option would be as simple putting se preexisting code together, and adding an entry to the setting page.

**Comment by patrickrushton** [comment]
+1 for this. I'm using Ferdi on macOS High Sierra which does not sync "dark appearance" with Ferdi. So it would be good to be able to schedule dark mode in Ferdi directly. 

On other apps like Things and PDF Expert, I can script the appropriate menu bar commands to switch dark mode via Applescript, but unfortunately Ferdi names the the menu bar item "Toggle Dark Mode" so this isn't possible because there is no way to know what mode Ferdi is currently in. If the menu bar item was instead named "Change to dark mode" and "Change to light mode" then this would work.
